core/src/apps/cardano/helpers/purposes.py

```diff
@@ -0,0 +1,4 @@
+from apps.common import HARDENED
+
+BYRON = 44 | HARDENED
+SHELLEY = 1852 | HARDENED
```

Comment:
Could you link me to a document where the 1852' purpose is documented?

Reply:
https://github.com/input-output-hk/implementation-decisions/pull/18 this was inherited from Jormungandr

Reply:
Thanks